client/src/components/Address/Address.js

```diff
@@ -41,7 +74,65 @@ const Address = () => {
         setMessage("Sorry! Something went wrong. Please Try again", err);
       });
   }, []);
+  const displayRazorPay = async () => {
+    const res = await loadRazorPay(
+      "https://checkout.razorpay.com/v1/checkout.js"
```

Comment:
URLs cannot be added like this in code directly. Put it in .env or config file

Reply:
I have moved